collections/_infographics/policies/emissions-pricing-world.md

```diff
@@ -13,4 +13,56 @@ data-orig:
   - [ "WBG Report", "https://openknowledge.worldbank.org/handle/10986/31755" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/zpoplatneni-emisi-svet" %}
+## How to read the infographic?
+
+This infographic shows the **total emissions for a country** or region, expressed as the size of a circle. The coloured box indicates the **amount of emissions covered by a carbon tax or an emission allowance scheme**, and the colour, or combination of colours, corresponds to the **type of measures implemented**. The symbols illustrate the **economic sectors affected by the measures**.
+
+The area of the largest circle (China) corresponds to 12.4 Gt <glossary id="co2eq">CO<sub>2</sub>eq</glossary> and the size of the smallest circle (Switzerland) to 0.05 Gt CO<sub>2</sub>eq. Individual measures have been divided into emission trading systems (ETS) and carbon tax, depending on how they effectively work. **Approaches to carbon pricing have evolved in recent years** and some cannot be directly assigned to one of the categories described here. The authors acknowledge that a different categorisation may apply.
+
+### More details on individual countries
+{% comment %}
+{% include preview-box.html
+    title="European emission allowances"
+    slug="emisni-povolenky-ets"
+    text="For a more detailed explanation of how the EU Emissions Trading Scheme works, see our explainer."
+%}
+{% endcomment %}
+* Canadian measures include both components of the federal system (ETS and carbon tax) as well as provincial measures that charge for carbon emissions.
+* The RGGI (Regional Greenhouse Gas Initiative) was the first greenhouse gas pricing measure in the United States and now involves ten northeastern states.
+* The European Emissions Trading Scheme also includes Norway, Iceland and Liechtenstein. In addition to an across-the-board ETS, some EU countries have implemented a carbon tax.
+* Japan's system includes the Satiama Prefecture ETS and the Tokyo ETS.
+* China has eight pilot emission trading systems in operation so far, with a national ETS to be launched in 2020. The coverage is based on preliminary unofficial estimates from an announcement by the State Development and Reform Commission of the People's Republic of China. The national ETS will initially cover only the power sector, with other sectors covered by individual pilot emission trading systems.
+
+## The underlying study of the infographic
```

Comment:
This translation is vague. Let's change the whole headline to "The Study Used in This Infographic", otherwise it can sound like we are studying the infographic.